ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/api/FieldCollection.java

```diff
@@ -191,6 +196,19 @@
       return this;
     }
 
+    FieldCollectionBuilder<Type, StateType, Parent> field(TypedPropertyGetter<Type, ?> getter,
+        DisplayContext context, String showFirstCondition, String showSecondCondition, boolean showSummary) {
+      if (null != showFirstCondition && null != rootFieldname) {
+        showFirstCondition = showFirstCondition.replace("{{FIELD_NAME}}", rootFieldname);
+      }
+      if (null != showSecondCondition && null != rootFieldname) {
+        showSecondCondition = showSecondCondition.replace("{{FIELD_NAME}}", rootFieldname);
+      }
+      field(getter).context(context).showCondition(showFirstCondition).showCondition(showSecondCondition)
```

Comment:
won't the second show condition always override the first show condition here ?

Could you please give an example where this will be used ?

Reply:
Ah right, I thought the conditions would be added on separately. 

It is a very minor problem but there is a scenario on the Solicitor-createSolAboutThePetitionerPage where if the Solicitor clicks 'Yes the petitioner has changed their name' and then clicks on 'Other', and then changes the answer to 'No the petitioner hasn't changed their name', the 'Other' box remains on screen and needs filling out. 
So I was trying to make the Other box need 2 conditions in order to stay on the screen.

Reply:
```
boolean showCondition = showFirstCondition && showSecondCondition;
field(getter).context(context).showCondition(showFirstCondition).showSummary(showSummary);
```
may be something like above might work ?

Reply:
I think the showCondition needs to be a string. I've tried adding the string conditions together, but don't think that's going to work either unfortunately